### PR TITLE
Allow MobileMapListing to display text on no match

### PR DESCRIPTION
**Pull request by nray621**
feat(react-ui-ag): Allow MobileMapListing to display text on no match

    affects: @rentpath/react-ui-ag

### CODE REVIEW

File: packages/react-ui-core/src/Listing/ListingComponents.js [hunk 1]
```diff
@@ -66,9 +66,13 @@ Ratings.contextTypes = {
   rating: PropTypes.object,
 }
 
-const Photos = (props, { photos }) => (
-  photos ? <PhotoCarousel {...props} items={photos} /> : null
-)
+const Photos = (props, { photos }) => {
```
Comment (nray621) [inline packages/react-ui-core/src/Listing/ListingComponents.js]: This looks to do exactly the same thing. Honestly just curious why this is preferable to the ternary

File: stories/theme/ag/small/MobileListing.css [hunk 2]
```diff
@@ -23,6 +23,22 @@
...
+  max-width: 65%;
+  margin: 0 auto;
+  background: rgba(0, 0, 0, 0.5);
+  color: #FFFFFF;
+  border-radius: 5px;
+  padding: 0 6px;
+  text-align: center;
+  font-size: 12px;
+  font-weight: 700;
+  left: 50%;
```
Comment (nray621) [inline stories/theme/ag/small/MobileListing.css]: Not a big deal at all, but it'd be nice if we logically group attributes to make it easier to read. 
ex - position followed by top followed by left etc.

File: packages/react-ui-ag/src/Listings/MobileListing.js [hunk 3]
```diff
@@ -41,14 +39,19 @@ export default class MobileListing extends PureComponent {
...
-        {!categoryMatch && <div className={theme.MobileListing_NoMatch}>{noMatchText}</div>}
-        {rating && <ListingComponents.Ratings data-tid="ratings" {...ratings} />}
-        {sponsored && <Text className={theme.MobileListing_Sponsored}>{sponsoredText}</Text>}
-      </div>
+
+    return (
+      <React.Fragment>
+        {noMatchText && <Text className={theme.MobileListing_NoMatchText}>{noMatchText}</Text>}
+        {!singleFamily &&
+          <div className={theme.MobileListing_RatingAndSponsored}>
```
Comment (nray621) [inline packages/react-ui-ag/src/Listings/MobileListing.js]: I hate to add more conditionals here, but it looks like the `MobileListing_RatingAndSponsored` div is going to be rendered regardless of either of its children existing. I might be wrong, but from what I've seen its fairly common for neither to exist, and we could shave a off a DOM node.

File: packages/react-ui-ag/src/Listings/MobileListing.js [hunk 4]
```diff
@@ -21,7 +20,6 @@ export default class MobileListing extends PureComponent {
     theme: {},
     listing: {},
     ratings: {},
-    categoryMatch: true,
```
Comment (nray621) [inline packages/react-ui-ag/src/Listings/MobileListing.js]: 👍

**Comment by nray621** [comment]
I approve of this code. LOL. I can't actually approve since it was originally my PR though, which is appropriate. So it'll need one more review

**Comment by bethstevic** [comment]
Looks good to me

**Pull request merged**